scripts/checks.sh

```diff
@@ -409,6 +408,34 @@ function check_disk_expansion()
 	fi
 }
 
+function check_image_corruption()
+{
+	local images
+	local corrupted=()
+	local -i corrupted_count=0
+	# TODO: this command is filtering out any images with a size <1Kb (for delta-based images)
+	images=$(${TIMEOUT_CMD} ${ENG} image ls --format "{{.ID}} {{.Size}}" | awk '/![1-9]*B/{next;} {print $1}' | sort -u)
```

Comment:
I couldn't figure out a nice way to dehumanize these values from the engine directly.

Reply:
fwiw delta images have the labels `io.resin.delta.config` and `io.resin.delta.base` and you might be able to filter for them with the `--filter` flag

cc @dfunckt I think this might only be valid for the engine-based delta's though

Reply:
Yes @robertgzr, that is correct.